build-ffmpeg

```diff
@@ -209,7 +209,7 @@ if build "libvpx"; then
 fi
 
 if build "lame"; then
-	download "http://kent.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz" "lame-3.100.tar.gz"
+	download "https://sourceforge.net/projects/lame/files/lame/3.100/lame-3.100.tar.gz/download" "lame-3.100.tar.gz"
```

Comment:
The download URL doesn't take me to the download directly, instead it shows me a download page. I changed it to https://netcologne.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz to support https.